crates/interpreter/src/instructions/opcode.rs

```diff
@@ -498,6 +498,28 @@ impl OpCode {
     }
 }
 
+/// Returns true if the opcode modifies memory.
+/// <https://bluealloy.github.io/revm/crates/interpreter/memory.html#opcodes>
+/// <https://github.com/crytic/evm-opcodes>
+#[inline]
+pub const fn modifies_memory(opcode: OpCode) -> bool {
```

Comment:
this should just be a function of the opcode type instead

Reply:
sure, I moved the method into the struct.